src/backtrace/dbghelp64.rs

```diff
@@ -127,5 +130,9 @@ pub unsafe fn trace(cb: &mut dyn FnMut(&super::Frame) -> bool) {
             &mut establisher_frame,
             ptr::null_mut(),
         );
+
+        if context.ip() == 0 || (context.ip() == previous_ip && context.sp() == previous_sp) {
+            break;
+        }
```

Comment:
> ### `[out] HandlerData`
>
> The location of the PC. If this parameter is 0, the PC is in the prologue, epilogue, or a null frame region of the function. If this parameter is 1, the PC is in the body of the function.
>
> **This parameter is not present on x64.**
>
> ### `[out] EstablisherFrame`
>
> A pointer to a FRAME_POINTERS structure that receives the establisher frame pointer value. The real frame pointer is defined only if InFunction is 1.
>
> **This parameter is of type PULONG64 on x64.**

Hmm.

Are our bindings for this function correct on all platforms...?

Reply:
The documentation is incorrect - `HandlerData` exists on x64.

ARM, ARM64 and x64 all have the same number of parameters with the same names, and the types of the parameter types are identical between ARM64 and x64.